draft-ietf-dprive-dnsoquic.md

```diff
@@ -569,11 +554,27 @@ connections. For example:
 
 Servers and Clients manage flow control as specified in QUIC.
 
-Servers MAY use the "maximum stream ID" option of the QUIC transport to limit
+Servers MAY use the "maximum stream ID" parameter of the QUIC transport to limit
```

Comment:
The "MAY" here is unnecessary (it's a restatement of QUIC requirements).

"can" is fine.